docs/dotnet-installation.md

```diff
@@ -32,23 +59,26 @@ All tests will be instrumented automatically.
 
 The following environment variables need to be configured when instrumenting your tests or application:
 
-| Environment variable  | Default value    | Description                                            |
-|-----------------------|------------------|--------------------------------------------------------|
-| `$SCOPE_APIKEY`       |                  | API key to use when sending data to Scope              |
-| `$SCOPE_API_ENDPOINT` |                  | API endpoint of the Scope installation to send data to |
-| `$SCOPE_COMMIT_SHA`   | Autodetected (*) | Commit hash to use when sending data to Scope          |
-| `$SCOPE_REPOSITORY`   | Autodetected (*) | Repository URL to use when sending data to Scope       |
-| `$SCOPE_SOURCE_ROOT`  | Autodetected (*) | Repository root path                                   |
+| Environment variable  | Default value           | Description                                            |
+|-----------------------|-------------------------|--------------------------------------------------------|
+| `$SCOPE_DSN`          |                         | API Endpoint and API Key in the form `https://<SCOPE_APIKEY>@<SCOPE_API_ENDPOINT>` to use when sending data to Scope (*)              |
```

Comment:
I wouldn't say that the DSN has that form - for the user, it should be a opaque token to be copy pasted as-is

Reply:
done